jupyterlab_caip_optimizer/src/components/suggest_trials/index.tsx

```diff
@@ -0,0 +1,120 @@
+/**
```

Comment:
WDYT about moving the bottom buttons above the table? In case the list of suggested trials is long